scripts/replace.js

```diff
@@ -47,6 +47,7 @@ function replaceWindow(aWindow, aSubstitutionList, aPrefs) {
       case group.HTML_NONE: replaceText(doc, consecutiveGroups, aPrefs); break;
       case group.HTML_OUTPUT: replaceTextWithHtml(doc, consecutiveGroups, aPrefs); break;
       case group.HTML_INPUT_OUTPUT: replaceHtml(doc, consecutiveGroups, aPrefs); break;
+      default: console.error('Fox Replace: Invalid output type');
```

Comment:
I don't think this is necessary, but in any case it is FoxReplace, without space, and it would be the HTML mode that is wrong, not the output type. But since there are no other checks like this in this file, I would remove the line entirely.